akka-actor/src/main/scala/akka/event/LoggerMailbox.scala

```diff
@@ -43,6 +43,5 @@ private[akka] class LoggerMailbox(owner: ActorRef, system: ActorSystem)
         envelope = dequeue
       }
     }
-    super.cleanUp(owner, deadLetters)
```

Comment:
There is a comment about this in the LoggerMailbox, otherwise we would not dare to do the loop without limits.

I think it's a good practice to call super in cleanup code, you never know if something more is added to the super class later.